packages/@adobe/spectrum-css-temp/components/searchwithin/index.css

```diff
@@ -11,49 +11,55 @@ governing permissions and limitations under the License.
 */
 
 @import '../commons/index.css';
-
-:root {
-  --spectrum-searchwithin-width: 250px;
+.spectrum-SearchWithin {
+  --spectrum-searchwithin-width: 240px;
+  --spectrum-searchwithin-searchfield-width: 128px;
 
   /* Force override */
   --spectrum-searchwithin-border-radius: 0;
-  --spectrum-searchwithin-dropdown-min-width: 0;
+  --spectrum-searchwithin-picker-min-width: 0;
 }
 
 .spectrum-SearchWithin {
-  width: var(--spectrum-searchwithin-width);
+  inline-size: var(--spectrum-searchwithin-width);
   display: inline-flex;
   position: relative;
+}
 
-  .spectrum-Dropdown {
-    width: auto;
-    min-width: 0;
-  }
+.spectrum-SearchWithin-picker {
+  inline-size: auto;
+  min-inline-size: 0;
```

Comment:
Even though they didn't give us a value, i think we should use `--spectrum-global-dimension-size-900` for the min-inline-size, that way it doesn't do what it's doing in the 30px story

Reply:
<img width="141" alt="Screen Shot 2021-07-27 at 4 24 54 PM" src="https://user-images.githubusercontent.com/698229/127240073-800f6da2-25a1-45af-bde8-e9603c7f80e3.png">

Reply:
Great idea, adding now.